app/models/project.server.ts

```diff
@@ -1,8 +1,10 @@
-import type { Profiles } from "@prisma/client";
+import type { Profiles, Projects } from "@prisma/client";
 import { Prisma } from "@prisma/client"
 
 import { prisma as db } from "~/db.server";
 
+export type { Projects } from "@prisma/client";
```

Comment:
I think this type is not used in the client side, why are we exporting it?

Reply:
You are right, instead I later added the new type `ProjectComplete` extracted from the getProject function.